Assets/NewBehaviourScript.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class NewBehaviourScript : MonoBehaviour
```

Comment:
I deleted this file for you. Be sure to check before you add everything in a commit! ;P